lib/ui/discover/discover_child_item.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:farmsmart_flutter/data/model/article_entity.dart';
+import 'package:farmsmart_flutter/ui/myplot/my_plot_item_footer.dart';
+import 'package:farmsmart_flutter/utils/box_shadows.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:farmsmart_flutter/utils/dimens.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:farmsmart_flutter/utils/styles.dart';
+import 'package:flutter/material.dart';
+import 'package:farmsmart_flutter/utils/assets.dart';
+
+
+class DiscoveryListItem {
+  Widget buildListItem(ArticleEntity articleData, goToArticleDetail) {
+    return GestureDetector(
+        onTap: () {
+          goToArticleDetail(articleData);
+        },
+        child: Padding(
+            key: ValueKey(articleData.title ?? "No Title"),
+            padding: Margins.boxPadding(),
+            child: Container(
+                height: 90,
+                decoration: BoxDecoration(
+                    color: Colors.white,
```

Comment:
Please refer to the colors common file or colors class

Reply:
that color property has been deleted to approach the view to the deign pattern, but yes it should be refer to our colors common file, Thnks !!